apiserver/facades/client/applicationoffers/access_test.go

```diff
@@ -42,7 +42,7 @@ func (s *offerAccessSuite) SetUpTest(c *gc.C) {
 	var err error
 	thirdPartyKey := bakery.MustGenerateKey()
 	s.authContext, err = crossmodel.NewAuthContext(
-		s.mockState, thirdPartyKey,
+		s.mockState, names.NewModelTag("uuid"), thirdPartyKey,
```

Comment:
can we please use `github.com/juju/juju/core/model/testing` and generate a new uuid and use valid uuids ?

My personal thinking is that we are testing correctness the test should be correct as well.